src/components/Layout/Navbar.vue

```diff
@@ -11,9 +11,28 @@
         <v-btn id="nav-user" text to="/user">Account</v-btn>
         <v-btn id="nav-logout" text to="/logout">Logout</v-btn>
       </template>
-      <template v-if="!isLoggedIn">
-        <v-btn id="nav-login" text to="/login">Log in</v-btn>
-        <v-btn id="nav-create-account" ext color="primary" to="/create-account">Sign up</v-btn>
+      <template v-else-if="$vuetify.breakpoint.xs && !isLoggedIn">
```

Comment:
My bad for not catching this in the last review where I did not see that there is a `template` block above.

Would it be possible to make the conditions simpler by using:
```
<template v-if="isLoggedIn` />
<template v-else-if="$vuetify.breakpoint.xs" />
<template v-else />
```

Maybe that's a bit easier to understand?